src/EnergyPlus/RootFinder.cc

```diff
@@ -439,7 +439,7 @@ namespace RootFinder {
     void IterateRootFinder(RootFinderDataType &RootFinderData, // Data used by root finding algorithm
                            Real64 const X,                     // X value of current iterate
                            Real64 const Y,                     // Y value of current iterate
-                           Optional_bool IsDoneFlag            // If TRUE indicates that the iteration should be stopped
+                           bool &IsDoneFlag                    // If TRUE indicates that the iteration should be stopped
```

Comment:
This argument wasn't really optional. Tripped on that during unit test development.

Reply:
Yay, less optionals!